### PR TITLE
docs: add marketplace change to changelog

### PR DESCRIPTION
**Pull request by code-asher**

### CODE REVIEW

**Review by jsjoeio — APPROVED** [review]

**Comment by code-asher** [comment]
Ah whoops accidentally deleted a section, will fix.

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,14 +22,6 @@ VS Code v99.99.999
 
 ## [Unreleased](https://github.com/cdr/code-server/releases)
 
-VS Code v0.00.0
-
-### Changed
-
-- Add here
-
-## [4.0.1](https://github.com/cdr/code-server/releases/tag/v4.0.1) - 2022-01-04
```
Comment (jsjoeio) [inline CHANGELOG.md:31]: Wait did you intend to delete this?

Comment (jsjoeio) [reply CHANGELOG.md:31]: Ah I see above

Comment (code-asher) [reply CHANGELOG.md]: Fixed!

**Review by jsjoeio — APPROVED** [review]

**Pull request merged**